ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed under the MIT License.
+
+namespace DataAccessLibrary
+{
+    using Microsoft.Data.Sqlite;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Windows.Storage;
+
+    /// <summary>
+    /// Data access class for the local User table in SQLite database
+    /// </summary>
+    public static class DataAccess
+    {
+        /// <summary>
+        /// Initializes the database, which is called when application starts
+        /// </summary>
+        public async static void InitializeDatabase()
```

Comment:
`IntializeDatabaseAsync`. All async methods should be suffixed by `Async` as per convention, to make it clear to callers that they need to be `await`ed.